README.md

```diff
@@ -175,17 +169,19 @@ Go to definition can be used to navigate to both parameter and variable definiti
 
 ![Image showing the Go to Definition dialog on a parameter reference](./images/go-to-definition.png)
 
+### Find all references
+
+Right-click on a parameter, variable, or user-defined function name and select **Go to References**. Double-clicking on any reference moves your cursor to the reference location.
+
+![Image showing the go to reference user interface](./images/parameter-references.png)
+
 ## Completions
 
 The ARM Tools extension provides code completions for the following:
 
-  - [Built-in functions](https://go.microsoft.com/fwlink/?LinkID=733958)
+  - [Built-in functions such as resourceId and variable](https://go.microsoft.com/fwlink/?LinkID=733958)
```

Comment:
variable?  Vars/params are mentioned specially in the next couple of lines.

Reply:
Fixed, thanks.